iot_api/user_api/resources/Inventory.py

```diff
@@ -270,19 +270,22 @@ def get(self, asset_type, asset_id):
                 until=until,
                 alert_types=[AlertType.find_one(alert_type_code).id for alert_type_code in alert_types],
                 devices=[asset.id],
+                alert_asset_type = AlertAssetType.DEVICE,
```

Comment:
I think it is better if repository functions receives basic types. Here, alert_asset_type could be a string in ("device", "gateway", None). Inside the function the string is replaced by the corresponding enum type. If the parameter given is not in ("device", "gateway", None) an Error.BadParameter should be thrown.

Reply:
I agree, done!